_rules/links-identical-name-equivalent-purpose-b20e66.md

```diff
@@ -154,19 +161,96 @@ A set of two SVG `<a>` elements have the same [accessible name][] and link to th
 </svg>
 ```
 
+#### Passed Example 10
+
+A set of one SVG `a` element and one HTML `a` element that have the same [accessible name][] and link to the [same resource][].
+
+```html
+<a href="http://facebook.com">Follow us"</a>
+
+<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+	<a href="http://facebook.com" aria-label="Follow us">
+		<circle cx="50" cy="40" r="35" />
+	</a>
+</svg>
+```
+
+#### Passed Example 11
+
+All three links have the same [accessible name][]. The second link ("from the light") is only part of the [light tree][]. When the [shadow tree][] is attached to `host` and flattened, this link is overwritten and therefore not part of the [flat tree][]. Hence, only the first and third link are considered by this rule and they both point to the [same resource][].
+
+```html
+<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Contact us</a>
+all the time.
+
+<div id="host">
+	<span
+		><a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html"
+			>Contact us</a
+		>
+		from the light.</span
+	>
+</div>
+
+<script>
+	const host = document.getElementById('host')
+	const shadowRoot = host.attachShadow({ mode: 'open' })
+
+	shadowRoot.innerHTML =
+		'<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Contact us</a> from the shadow.'
+</script>
+```
+
+#### Passed Example 12
```

Comment:
IMO one shadow DOM example is enough. We're not testing shadow DOM implementations, we're testing links.